R/create.clone.genome.distribution.scatterplot.R

```diff
@@ -0,0 +1,35 @@
+create.clone.genome.distribution.scatterplot <- function(
+    scatter.df,
+    nsnv,
+    nclone,
+    chr.info,
+    save.plt = NULL,
+    xlab.label = '',
+    xaxis.tck = 0,
+    xaxis.cex = 0,
+    ...
+    ) {
+
+    return(BoutrosLab.plotting.general::create.scatterplot(
+        filename = save.plt,
+        formula = clone.id ~ genome.pos,
+        data = scatter.df,
+        xlab.top.lab = paste0('Total SNVs: ', nsnv),
+        xlab.top.y = 0.5,
+        xlab.label = xlab.label,
+        ylab.label = 'Clone ID',
+        xaxis.lab = chr.info$chr,
+        yaxis.lab = rev(levels(scatter.df$clone.id)),
```

Comment:
Labels are reversed. This line needs changing.
```suggestion
        yaxis.lab = levels(scatter.df$clone.id),
```